test/unit/argument_parser/format_parse_validators_test.cpp

```diff
@@ -209,31 +209,56 @@ TEST(validator_test, output_file)
     { // single file
 
         { // empty list of file.
-            seqan3::output_file_validator my_validator{};
+            seqan3::output_file_validator my_validator{seqan3::output_file_open_options::create_new};
             EXPECT_NO_THROW(my_validator(tmp_name.get_path()));
         }
 
-        { // file does not exist.
+        { // file does not exist
+            remove(tmp_name_2.get_path());
```

Comment:
???? Where does this function come from?

Do you mean

```
            std::filesystem::path not_existing_path = tmp_name_2.get_path();
            not_existing_path.remove_filename();
```

Reply:
but shouldn't this not exist anyway?  why do you need to remove it?

Does seqan3::test::tmp_filename create a file?